demod.c

```diff
@@ -86,9 +86,12 @@ void mdcCallBack(int numFrames, unsigned char op, unsigned char arg, unsigned sh
                                          "\"ex0\":\"%02x\","\
                                          "\"ex1\":\"%02x\","\
                                          "\"ex2\":\"%02x\","\
-                                         "\"ex3\":\"%02x\"}", (int)time(NULL), op, arg, unitID,extra0, \
```

Comment:
Good catch - `time(NULL)` would have reflected the time + however long it took to decode